examples/alanine_dipeptide_tps/AD_tps_1b_trajectory.ipynb

```diff
@@ -87,9 +90,7 @@
   {
    "cell_type": "code",
    "execution_count": 5,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
```

Comment:
Line `112` should use 
`print(trajectories)` 
instead of
`print trajectories`